modules/openapi-generator/src/main/resources/php-symfony/composer.mustache

```diff
@@ -19,22 +19,24 @@
         }
     ],
     "require": {
-        "php": "^7.1.3",
+        "php": ">=8.0.2",
         "ext-curl": "*",
         "ext-json": "*",
         "ext-mbstring": "*",
-        "symfony/validator": "*",
-        "jms/serializer-bundle": "^2.0",
-        "symfony/framework-bundle": "^4.4.8"
+        "symfony/validator": "6.0.*",
+        "jms/serializer-bundle": "6.0.*",
```

Comment:
Latest available version is 4.0.2
https://packagist.org/packages/jms/serializer-bundle